src/app/dashboard/page.jsx

```diff
@@ -62,9 +62,9 @@ export default function page() {
           </div>
         </div>
         <Tabs />
-        <div className="h-full flex border-2 border-[#FFFFFF20] rounded-lg">
-          <div className="flex flex-col basis-1/3 justify-between py-8 pl-10">
-            <div className="">
+        <div className="h-full flex flex-col lg:flex-row border-2 border-[#FFFFFF20] rounded-lg">
```

Comment:
Almost all `flex-col lg:flex-row` in the whole code can be replaced with `max-lg:flex-col`. Try doing that, and check the results!!